pkg/karmadactl/util/check.go

```diff
@@ -60,9 +60,9 @@ func WaitForStatefulSetRollout(c kubernetes.Interface, sts *appsv1.StatefulSet,
 }
 
 // WaitForDeploymentRollout  wait for Deployment reaches the ready state or timeout.
-func WaitForDeploymentRollout(c kubernetes.Interface, dep *appsv1.Deployment, timeoutSeconds int) error {
+func WaitForDeploymentRollout(c kubernetes.Interface, dep *appsv1.Deployment, duration time.Duration) error {
```

Comment:
```suggestion
func WaitForDeploymentRollout(c kubernetes.Interface, dep *appsv1.Deployment, timeout time.Duration) error {
```
timeout or timeoutDuration